spec/controllers/api/v1/bikes_controller_spec.rb

```diff
@@ -69,7 +69,7 @@
             rear_wheel_size: nil,
             rear_gear_type_slug: nil,
             handlebar_type_slug: nil,
-            frame_material_slug: nil,
```

Comment:
@gadogado this really, **really** shouldn't have happened. 

I'm sorry that I missed it